openwisp_radius/base/models.py

```diff
@@ -1238,7 +1239,7 @@ class AbstractOrganizationRadiusSettings(UUIDModel):
         blank=True,
         max_length=200,
         help_text=_PASSWORD_RESET_URL_HELP_TEXT,
-        default=DEFAULT_PASSWORD_RESET_URL,
+        default=None,
```

Comment:
we should be able to remove `null`, `blank` and `default` now from every fallback field.